tutorials/containers/introduction-to-lxd-projects/introduction-to-lxd-projects.md

```diff
@@ -0,0 +1,417 @@
+---
+id: introduction-to-lxd-projects
+summary: An introduction to the projects feature in LXD
+categories: containers
+tags: tutorial,desktop,server,lxd,lxc,projects
+difficulty: 3
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-04-24
+author: Thomas Parrott <thomas.parrott@canonical.com>
+
+---
+
+# Introduction to LXD projects
+
+## Overview
+Duration: 5:00
+
+[LXD](https://linuxcontainers.org/lxd) is a container hypervisor providing a ReST API to manage LXC containers.
+It provides a virtual machine like experience without incurring the overhead of a traditional hypervisor.
+
+
+![logo](images/containers.png)
+
+However when you are managing lots of containers providing different services, it can become confusing to see which containers are dependent on each other.
+
+The projects feature is designed to help in this situation by providing the ability to group one or more containers together into related "projects" that can be used with the `lxc` tool.
+
+This tutorial will show how to create LXD projects, add containers into a project and explore the features that can be specified at a project level.
+
+### Requirements
+
+  - LXD installed and running on Ubuntu 18.04 or higher
+  - You should know how to create and launch an LXD container
+
+## Creating our first project
+Duration: 5:00
+
+Imagine we have a scenario where we are setting up a web site for a particular client.
+We want to create two containers; one for the web service and one for the database service.
+Because we have lots of different web sites for different clients we want to ensure that we can quickly see and manage all of the containers for this particular web site.
+
+projects help us in this situation because we can create a project called `client-website` and add the containers to it.
+
+```bash
+lxc project create client-website -c features.images=false -c features.profiles=false
+```
+
+In the above command we have passed the `-c features.images=false` and `-c features.profiles=false` flags so that our new project will still use the images and profiles of the `default` project and will not namespace them (we'll come onto that some more later in the tutorial).
+
+Now we switch the `lxc` tool into that project.
+
+```bash
+lxc project switch client-website
+```
+
+Let's check that we have switched to the new empty project:
+
+```bash
+lxc ls
+```
+
+We would expect to see an empty list at this point.
+
+We assume that you have already installed and configured LXD. If not, complete the [“Setting up LXD on Ubuntu 16.04”](/tutorial/tutorial-setting-up-lxd-1604) tutorial.
```

Comment:
Ah, good but this probably should be moved earlier then?

Reply:
@stgraber does the LXD version in Ubuntu 16.04 support projects? Shall I just add a section for snap install instead (or update the existing guide to the latest 18.04 with snap?).

Reply:
The LXD snap works on Ubuntu 14.04 and higher, so as long as the instructions say to use the snap, they will have project support.

Reply:
@stgraber its quite out of date sadly, still uses apt. Shall I add a basic LXD setup guide to this article or refer to that one and update it with snap?